metricflow/model/parsing/schemas/metricflow.json

```diff
@@ -0,0 +1,690 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "definitions": {
+        "composite_sub_identifier_schema": {
+            "$id": "composite_sub_identifier_schema",
+            "additionalProperties": false,
+            "properties": {
+                "expr": {
+                    "type": [
+                        "string",
+                        "boolean"
+                    ]
+                },
+                "name": {
+                    "type": "string"
+                },
+                "ref": {
+                    "type": "string"
+                }
+            },
+            "type": "object"
+        },
+        "data_source": {
+            "$id": "data_source",
+            "additionalProperties": false,
+            "properties": {
+                "constraint": {
+                    "$ref": "constraint_schema"
```

Comment:
This schema doesn't exist. I thought @QMalcolm removed it. Did you run this on an old branch?

Reply:
ah, i must have. good catch

Reply:
Alright fixed it